searches/jump_search.py

```diff
@@ -21,6 +42,7 @@ def jump_search(arr, x):
 
 
 if __name__ == "__main__":
-    arr = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610]
-    x = 55
+    user_input = input("Enter numbers separated by a comma:\n").strip()
+    arr = [int(item) for item in user_input.split(",")]
+    x = int(input("Enter the number to be searched:\n"))
     print(f"Number {x} is at index {jump_search(arr, x)}")
```

Comment:
-1 is a valid position in python, check if the number is found before printing it